### PR TITLE
Clarify Kubernetes Support Lifecycle

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds information about our support life cycle for Kubernetes.

## Review Changes

💻 [Preview URL](https://deploy-preview-2013--docs-spectrocloud.netlify.app/integrations/kubernetes-support)

🎫 [DOC-1014](https://spectrocloud.atlassian.net/browse/DOC-1014)

### CODE REVIEW

File: docs/docs-content/integrations/kubernetes-support.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+We support CNCF Kubernetes, [Palette eXtended Kubernetes (PXK)](./kubernetes.md) and [PXK-E](./kubernetes-edge.md) for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. 
+
+The diagram below illustrates the support lifecycle of a Kubernetes version. A deprecated Kubernetes version will no longer receive updates.
+
+![Diagram of the Kubernetes Support Lifecycle. PXK and CNCF is supported for 14 months. After that, normal deprecation flow is initiated.](/integrations_kubernetes-support_support-cycle.png)
+ 
+Additional support may be extended for PXK and PXK-E. Discuss this with our support team if you need extended support. You can contact our support team at [support@spectrocloud.com](mailto:support@spectrocloud.com).
+
+
+This is at spectrocloud support discretion and is not guaranteed.
```
Comment (addetz) [inline docs/docs-content/integrations/kubernetes-support.md:45]:
```suggestion
This is at Spectro Cloud support discretion and is not guaranteed.
```

File: docs/docs-content/integrations/kubernetes.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ We also offer Palette eXtended Kubernetes Edge (PXK-E) for Edge deployments. Ref
 
 :::
 
+### Support Lifecycle
+
+We support PXK for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](kubernetes-support.md#palette-extended-kubernetes-support) guide.
```
Comment (addetz) [inline docs/docs-content/integrations/kubernetes.md:39]:
```suggestion
We support PXK for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more in the [Kubernetes Support Lifecycle](kubernetes-support.md#palette-extended-kubernetes-support) guide.
```

File: docs/docs-content/integrations/microk8s.md [hunk 3]
```diff
@@ -12,6 +12,10 @@ tags: ['packs', 'microk8s', 'kubernetes']
 
 MicroK8s is a Cloud Native Computing Foundation (CNCF) certified upstream Kubernetes deployment that runs entirely on your workstation or edge device. It runs all Kubernetes services natively without virtual machines and packs all the required libraries and binaries.
 
+### Support Lifecycle
+
+We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```
Comment (addetz) [inline docs/docs-content/integrations/microk8s.md:17]:
```suggestion
We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. Learn more on the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```

File: docs/docs-content/integrations/rke2.md [hunk 4]
```diff
@@ -27,6 +27,11 @@ RKE2 launches control plane components as static pods, managed by the kubelet in
 You can deploy RKE2 by adding this pack to a cluster profile. Once the cluster profile is created, you can deploy the RKE2-based Kubernetes clusters through Palette.
 
 
+### Support Lifecycle
+
+We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```
Comment (addetz) [inline docs/docs-content/integrations/rke2.md:32]:
```suggestion
We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. Learn more on the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```

File: docs/docs-content/integrations/kubernetes-support.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Kubernetes follows the [semantic version schema](https://semver.org/). Versions are annotated as x.y.z, where x is the major version, y is the minor version, and z is the patch version.
+:::
+
+We support N-3 Kubernetes minor versions until the official End-Of-Life (EOL). Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our deprecation process in the [Pack Deprecation](./maintenance-policy.md#pack-deprecations) section. 
+
+
+## Palette eXtended Kubernetes Support
+
+We support CNCF Kubernetes, [Palette eXtended Kubernetes (PXK)](./kubernetes.md) and [PXK-E](./kubernetes-edge.md) for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process.
```
Comment (addetz) [inline docs/docs-content/integrations/kubernetes-support.md:36]:
```suggestion
We support CNCF Kubernetes, [Palette eXtended Kubernetes (PXK)](./kubernetes.md) and [Palette eXtended Kubernetes Edge (PXK-E)](./kubernetes-edge.md) for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. 
```

File: docs/docs-content/integrations/kubernetes-generic.md [hunk 6]
```diff
@@ -20,6 +20,10 @@ Review the [Maintenance Policy](maintenance-policy.md) to learn about pack updat
 
 <br />
 
+### Support Lifecycle
+
+We support CNCF Kubernetes for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](kubernetes-support.md#palette-extended-kubernetes-support) guide.
```
Comment (addetz) [inline docs/docs-content/integrations/kubernetes-generic.md:25]:
```suggestion
We support CNCF Kubernetes for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. Learn more in the [Kubernetes Support Lifecycle](kubernetes-support.md#palette-extended-kubernetes-support) guide.
```

File: docs/docs-content/integrations/k3s.md [hunk 7]
```diff
@@ -12,6 +12,11 @@ tags: ["packs", "kubernetes", "k3s", "edge"]
 
 K3s is a lightweight distribution of Kubernetes that is specifically designed for resource-constrained environments and Internet of Things (IoT) applications. Unlike standard Kubernetes with multiple binaries for its various components, K3s is packaged as a single binary with a lightweight storage backend and minimal external dependencies. For more information about K3s, refer to [K3s documentation](https://docs.k3s.io/). 
 
+
+### Support Lifecycle
+
+We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. You can learn more about our Kubernetes support lifecycle in the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```
Comment (addetz) [inline docs/docs-content/integrations/k3s.md:18]:
```suggestion
We support other Kubernetes distributions such as K3s, Microk8s, and RKE2 until their official EOL. The EOL is set by the respective owner. Once we stop supporting the minor version, we initiate the deprecation process. Learn more on the [Kubernetes Support Lifecycle](./kubernetes-support.md#other-kubernetes-distributions) page.
```

**Review by addetz — APPROVED** [review]
Tiny comments 🚀

File: docs/docs-content/integrations/kubernetes-support.md (hunk 1 above)
Comment (karl-cardenas-coding) [reply docs/docs-content/integrations/kubernetes-support.md:45]: OMG - this was a test sentence. Thank you so much for flagging it.

**Review by yuliiiah — APPROVED** [review]
I've made suggestions on structure, word usage, and some other small things. Hope it's helpful, and congrats on the doc 🎉

File: docs/docs-content/integrations/kubernetes-support.md [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+## Palette eXtended Kubernetes Support
+
+We support CNCF Kubernetes, [Palette eXtended Kubernetes (PXK)](./kubernetes.md) and [PXK-E](./kubernetes-edge.md) for N-3 Kubernetes minor versions for a duration of 14 months. The duration exceeds the official EOL by four months. Once we stop supporting the minor version, we initiate the deprecation process. 
+
+The diagram below illustrates the support lifecycle of a Kubernetes version. A deprecated Kubernetes version will no longer receive updates.
+
+![Diagram of the Kubernetes Support Lifecycle. PXK and CNCF is supported for 14 months. After that, normal deprecation flow is initiated.](/integrations_kubernetes-support_support-cycle.png)
+ 
+Additional support may be extended for PXK and PXK-E. Discuss this with our support team if you need extended support. You can contact our support team at [support@spectrocloud.com](mailto:support@spectrocloud.com).
```
[the remark this answers is not in the document]
Comment (yuliiiah) [reply docs/docs-content/integrations/kubernetes-support.md:42]: Hm, this could also be an admonition, depending on how much attention we want to call.

**Pull request merged**